pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -271,14 +269,13 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
 		updateCleanupPolicy = kubectl.RenameRollingUpdateCleanupPolicy
 	}
 	config := &kubectl.RollingUpdaterConfig{
-		Out:            out,
-		OldRc:          oldRc,
-		NewRc:          newRc,
-		UpdatePeriod:   period,
-		Interval:       interval,
-		Timeout:        timeout,
-		CleanupPolicy:  updateCleanupPolicy,
-		UpdateAcceptor: kubectl.DefaultUpdateAcceptor,
+		Out:           out,
+		OldRc:         oldRc,
+		NewRc:         newRc,
+		UpdatePeriod:  period,
```

Comment:
Yes we want to preserve existing defaults.
But not sure why will setting these fields here, make them percentage based?
As you said in another comment, there will be a way to set an absolute number. We should use that to set it to 1, which preserves the existing default.